docs/dev/data_validation.rst

```diff
@@ -0,0 +1,45 @@
+===============================================================================
+Data Validation
+===============================================================================
+The data validation tests are organized into datasource specific modules under
+``test/validate``. They test the quality and internal consistency of the data
+that is output by the PUDL ETL pipeline. Currently they only work on the full
+dataset, and do not have a ``--fast`` option. While it is possible to run the
+full ETL process and output it in a temporary directory, to then be used by the
+data validation tests, that takes a long time, and you don't get to keep the
+processed data afterward. Typically we validate outputs that we're hoping to
+keep around, so we advise running the data validation on a pre-generated PUDL
+SQLite database.
+
+To point the tests at already processed data, use the ``--live_pudl_db`` and
+``--live_ferc1_db`` options. The ``--pudl_in`` and ``--pudl_out`` options work
+the same as above. E.g.
```

Comment:
No clear on what "above" is here. Also I don't think `--pudl_in` or `--pudl_out` are mentioned before in the dev docs, so it would be good to clarify what they are here or like to where they are described in full.